gemfiles/common.rb

```diff
@@ -1,6 +1,8 @@
+# frozen_string_literal: true
 source 'https://rubygems.org'
 
 gem 'activerecord-jdbcmysql-adapter', '~> 1.2', platforms: :jruby
 gem 'byebug', platforms: :ruby
+gem 'rubocop', platforms: :ruby
```

Comment:
Rubocop has a history of breaking changes between minor releases. I think we should lock to `~> 0.49.1` for now.